src/CBLQuery_CAPI.cc

```diff
@@ -88,6 +88,12 @@ CBLResultSet* CBLQuery_CopyCurrentResults(const CBLQuery* query,
     return listener->resultSet().detach();
 }
 
+void CBLQuery_SetListenerCallbackDelay(int delayMS) noexcept {
+#ifdef DEBUG
+    ListenerToken<CBLQueryChangeListener>::setC4QueryObserverCallbackDelay(delayMS);
+#endif
+}
+
```

Comment:
this method is not exposed for swift unless is debug (including the definition). Can we have it the same here? 
```
#ifdef DEBUG
void CBLQuery_SetListenerCallbackDelay(int delayMS) noexcept { ... }
#endif
```

Reply:
The method needs to be exposed in the public symbols like the other methods in the private header file. I don’t want to have the public symbols points to non existing symbols. The private headers will not be available publicly and the method will do nothing in release build.